apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/PathAssertionsSpec.kt

```diff
@@ -2,12 +2,14 @@ package ch.tutteli.atrium.api.fluent.en_GB.jdk8
 
 import ch.tutteli.atrium.creating.Expect
 import ch.tutteli.atrium.specs.fun0
+import ch.tutteli.atrium.specs.fun1
 import ch.tutteli.atrium.specs.notImplemented
 import java.nio.file.Path
 
 class PathAssertionsSpec : ch.tutteli.atrium.specs.integration.PathAssertionsSpec(
     fun0(Expect<Path>::exists),
-    fun0(Expect<Path>::existsNot)
+    fun0(Expect<Path>::existsNot),
+    fun1(Expect<Path>::endsWith)
 ){
     @Suppress("unused", "UNUSED_VALUE")
     private fun ambiguityTest() {
```

Comment:
please add endsWith to the ambiguityTest

Reply:
I'll fix it myself

Reply:
I'm not sure about the purpose, but are you want something like those in the `ambiguityTest()` function

```kotlin

        val a3: Expect<Path> = notImplemented()
        val a4: Path = notImplemented()

        a3.endsWith(a4)
```

Reply:
The purpose is to see whether the signature of the method makes sense (as this is sometimes hard to see in the review). Your current implementation for instance fails the ambiguityTest:
```
a1.endsWith(Paths.get("a"))
a2.endsWith(Paths.get("a"))
```
But no worries I'll fix it myself unless you would like to fix it?